### PR TITLE
Zeek kit: Sync up EV naming

### PR DESCRIPTION
**Pull request by keith-smiley-gravwell**
This PR addresses https://github.com/gravwell/kits/issues/98

### CODE REVIEW

File: zeek/searchlibrary/022a1dcb-393f-4249-8d89-6170800baae7.query [hunk 1]
```diff
@@ -1,3 +1,3 @@
-tag=zeekhttp ax
+tag=zeekhttp ax "id.orig" "id.resp" method uri
```
Comment (john-floren-gravwell) [inline zeek/searchlibrary/022a1dcb-393f-4249-8d89-6170800baae7.query:1]: orig_h, resp_h

File: zeek/searchlibrary/022a1dcb-393f-4249-8d89-6170800baae7.query [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
 | regex -v -e method "^-|GET|HEAD|POST|PUT|DELETE|CONNECT|OPTIONS|PATCH|TRACE$"
-| table orig resp method uri
\ No newline at end of file
+| table "id.orig" "id.resp" method uri
```
Comment (john-floren-gravwell) [inline zeek/searchlibrary/022a1dcb-393f-4249-8d89-6170800baae7.query:3]: ditto

File: zeek/searchlibrary/5a61cd1c-a1bd-4430-9ad4-c9cf9b3fd7a5.query [hunk 3]
```diff
@@ -1 +1,4 @@
-tag=zeekdns ax resp | stats count by resp | geoip -r asn_db resp.ASNOrg | table resp ASNOrg count
\ No newline at end of file
+tag=zeekdns ax "id.resp_h"
+| stats count by "id.resp_h"
+| geoip -r asn_db resp.ASNOrg
```
Comment (john-floren-gravwell) [inline zeek/searchlibrary/5a61cd1c-a1bd-4430-9ad4-c9cf9b3fd7a5.query:3]: id.resp_h here

File: zeek/searchlibrary/b74b3006-8c12-429e-b9a9-97b8ba71a0d8.query [hunk 4]
```diff
@@ -1,7 +1,7 @@
-tag=zeekssh ax resp_host host_key != -
-| alias resp_host latestserver
-| stats unique_count(resp_host) as unique_hosts by host_key unique_count(host_key) by resp_host
+tag=zeekssh ax "id.resp_h" host_key != "-"
+| alias "id.resp_h" latestserver
+| stats unique_count("id.resp_h") as unique_hosts by host_key unique_count(host_key) by "id.resp_h"
 | eval ( !has(unique_hosts) || unique_hosts > 1 )
 | sort by host_key
 | geoip -r asn_db resp_host.ASNOrg
```
Comment (john-floren-gravwell) [inline zeek/searchlibrary/b74b3006-8c12-429e-b9a9-97b8ba71a0d8.query:6]: id.resp_h here

File: zeek/template/84123640-bf99-415d-b69c-de438c46027e.query [hunk 5]
```diff
@@ -1 +1,3 @@
-tag=zeekconn ax resp==%%IP%% orig resp_bytes != - | stats unique_count(orig) as "Active Clients" | numbercard "Active Clients"
\ No newline at end of file
+tag=zeekconn ax "id.resp_h == "%%IP%%" "id.orig_h" resp_bytes != "-"
```
Comment (john-floren-gravwell) [inline zeek/template/84123640-bf99-415d-b69c-de438c46027e.query:1]: you're missing a closing " after id.resp_h

File: zeek/template/972a28a1-8310-4223-a49c-b7574d31fecb.query [hunk 6]
```diff
@@ -1,3 +1,3 @@
-tag=zeekconn ax resp==%%IP%% orig_bytes!="-" resp_bytes!="-" service !="-" |
-stats sum(orig_bytes) as upload sum(resp_bytes) as download |
-chart upload download
\ No newline at end of file
+tag=zeekconn ax "id.resp" == "%%IP%%" orig_bytes != "-" resp_bytes != "-" service != "-"
```
Comment (john-floren-gravwell) [inline zeek/template/972a28a1-8310-4223-a49c-b7574d31fecb.query:1]: id.resp_h here

**Review by john-floren-gravwell — CHANGES_REQUESTED** [review]

File: zeek/searchlibrary/022a1dcb-393f-4249-8d89-6170800baae7.query (hunk 1 above)
Comment (keith-smiley-gravwell) [reply zeek/searchlibrary/022a1dcb-393f-4249-8d89-6170800baae7.query]: Corrected

File: zeek/searchlibrary/022a1dcb-393f-4249-8d89-6170800baae7.query (hunk 2 above)
Comment (keith-smiley-gravwell) [reply zeek/searchlibrary/022a1dcb-393f-4249-8d89-6170800baae7.query]: Corrected

File: zeek/searchlibrary/5a61cd1c-a1bd-4430-9ad4-c9cf9b3fd7a5.query (hunk 3 above)
Comment (keith-smiley-gravwell) [reply zeek/searchlibrary/5a61cd1c-a1bd-4430-9ad4-c9cf9b3fd7a5.query]: Corrected

File: zeek/template/84123640-bf99-415d-b69c-de438c46027e.query (hunk 5 above)
Comment (keith-smiley-gravwell) [reply zeek/template/84123640-bf99-415d-b69c-de438c46027e.query]: Corrected

File: zeek/template/972a28a1-8310-4223-a49c-b7574d31fecb.query (hunk 6 above)
Comment (keith-smiley-gravwell) [reply zeek/template/972a28a1-8310-4223-a49c-b7574d31fecb.query]: Corrected

**Review by john-floren-gravwell — APPROVED** [review]

**Pull request merged**